buildchain/buildchain/salt_tree.py

```diff
@@ -453,16 +451,16 @@ def _get_parts(self) -> Iterator[str]:
     Path('salt/_modules/cri.py'),
     Path('salt/_modules/metalk8s_cordon.py'),
     Path('salt/_modules/metalk8s_drain.py'),
-    Path('salt/_modules/metalk8s_kubernetes.py'),
     Path('salt/_modules/metalk8s_etcd.py'),
     Path('salt/_modules/metalk8s_grafana.py'),
     Path('salt/_modules/metalk8s_kubernetes_utils.py'),
-    Path('salt/_modules/metalk8s.py'),
+    Path('salt/_modules/metalk8s_kubernetes.py'),
     Path('salt/_modules/metalk8s_network.py'),
     Path('salt/_modules/metalk8s_package_manager.py'),
-    Path('salt/_modules/metalk8s_volumes.py'),
+    Path('salt/_modules/metalk8s_solutions_k8s.py'),
     Path('salt/_modules/metalk8s_solutions.py'),
-
+    Path('salt/_modules/metalk8s_volumes.py'),
+    Path('salt/_modules/metalk8s.py'),
```

Comment:
That's a surprising reordering.

Reply:
Indeed, my editor shows them in this order, but I prefer the "usual" lexicographical ordering (such as the one for Python strings). I'll change this.